<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTeam</class>
 <widget class="QDialog" name="AddTeam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>153</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="SaveButton">
      <property name="text">
       <string>完成</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="NextButton">
      <property name="text">
       <string>下一组</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="LastButton">
      <property name="text">
       <string>上一组</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>SaveButton</zorder>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>241</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>组别:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="TypeBox">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>光电四轮组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>光电直立组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>电磁普通组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>电磁节能组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>光电追逐组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>电磁追逐组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>双车对抗组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四旋翼导航组</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>队名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TeamLine">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>编号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="NumberBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
